### PR TITLE
fix: file uploads

### PR DESCRIPTION
**Pull request by NarayanBavisetti**

### CODE REVIEW

File: apiserver/plane/api/views/asset.py [hunk 1]
```diff
@@ -73,3 +74,16 @@ def delete(self, request, asset_key):
             # Delete the file object
             file_asset.delete()
             return Response(status=status.HTTP_204_NO_CONTENT)
+
+class AssetsEndpoint(BaseAPIView):
+
+    permission_classes = [WorkspaceEntityPermission]
+    parser_classes = (MultiPartParser, FormParser)
+
+    def get(self, request, workspace_id, asset_key):
```
Comment (pablohashescobar) [inline apiserver/plane/api/views/asset.py:83]: Can we do a `slug` here instead of `workspace_id` ? @NarayanBavisetti

Comment (pablohashescobar) [reply apiserver/plane/api/views/asset.py:83]: Current permission layer works on `slug`

**Review by pablohashescobar — CHANGES_REQUESTED** [review]

File: apiserver/plane/api/views/asset.py (hunk 1 above)
Comment (NarayanBavisetti) [reply apiserver/plane/api/views/asset.py:83]: ok sure, will change to slug

**Comment by codevski** [comment]
Will this fix the issue how file uploads still direct to localhost?
image
image

**Pull request closed without merging**